### DISCUSSION TITLE
Are you familiar with Google Takeout?

### DISCUSSION THREAD

**Discussion by RonSijm** [gilesknap/gphotos-sync#430] [General]
Recently I wanted to switch from Google photos to a different cloud provider, and was looking for a way to get all my data out of google.

I came across this project ([gphotos-sync](https://github.com/gilesknap/gphotos-sync)) - but after some more googling, I also came across [google takeout](https://takeout.google.com/) - Which lets you download everything from Google Photos at once, at the original quality. (so it bypasses all the API limitations such as transcoding and downscaling of images)

If use-case is that you're actively syncing your photos from google cloud to somewhere else, the takeout doesn't seem the best option..

But if your use-case is just that you want to take everything out of google, the google takeout is pretty good

**Comment by gilesknap** [comment]
Yes I agree with your statement.

The only issue with Google Takeout is that the results are not prettily organized. Someone needs to write a Takeout organizer. I have considered it in the past but it would be a one off, because the Takeout process is laborious for a large library. Hence I'd only bother to do the work if I was planning on exiting Google for good. 

Despite the lock-in issues with the Google Photos API I'm still finding the service too good to exit.

**Reply by KjeldsenDK** [reply]
Google takeout is useless except for a disaster backup, there is zero support for albums.

**Reply by gilesknap** [reply]
I did not know that, I have not looked at it for a long time - my recollection was that it included duplicate files for all items in albums, but that may be a false memory.
